src/Http/Controller/AdvsController.php

```diff
@@ -756,7 +756,9 @@ public function view($seo, $id = null)
             if (!$adv->district_name) {
                 $metaDescTags = $adv->city_name.', ';
             }
-            $metaDescTags= $metaDescTags.$categories['cat1']['name'].' '.$categories['cat2']['name'].' '.trans('visiosoft.module.advs::field.adv_desc_metaTags');
+            $cat1_name=$categories['cat1']['name'] ?? null;
```

Comment:
why did you write `null` after the `(??)` nullish coalescing operator? this is already checking it is null or not .